addons/portal_rating/static/src/js/portal_rating_composer.js

```diff
@@ -97,13 +95,22 @@ publicWidget.registry.RatingPopupComposer = publicWidget.Widget.extend({
 
     /**
      * @private
-     * @param {OdooEvent} ev
+     * @param {OdooEvent} event
      */
-    _onReloadRatingPopupComposer: function (ev) {
-        this._reloadRatingPopupComposer(ev.data);
+    _onReloadRatingPopupComposer: function (event) {
+        const data = event.data;
+
+        // Refresh the internal state of the widget
+        this.rating_avg = data.rating_avg;
+        this.rating_count = data.rating_count;
+        this.rating_value = data.rating_value;
+
+        this.options = _.extend(this.options, data);
+
+        this._reloadRatingPopupComposer();
     }
 });
 
-return RatingPopupComposer;
+return publicWidget.registry.RatingPopupComposer;
```

Comment:
According to js guidelines: "Name all entities exported by a JS module." ( https://github.com/odoo/odoo/wiki/Javascript-coding-guidelines ) Shouldn't we keep the return here ?

Reply:
ok :)